### DISCUSSION TITLE
Byte Range Requests

### DISCUSSION THREAD

**Discussion by Gigamick** [katspaugh/wavesurfer.js#3662] [Q&A]
Wavesurfer has been such a great package for me, however my app is getting busier and busier and I'm getting more complaints about loading time for large wave files. 

Is wavesurfer compatible with a "byte range requests" implementation, such as like the web players for Spotify and Soundcloud etc have in place?

**Comment by katspaugh** [comment]
Short answer is no. Please check the FAQ in the readme.

**Reply by Gigamick** [reply]
Apologies, I'm not intentionally trying to be a d!ck but I can't see anything about byte range requests in the FAQ:

<img width="783" alt="Screenshot 2024-04-16 at 15 36 33" src="https://github.com/katspaugh/wavesurfer.js/assets/22120297/f3444871-6cea-46a8-a1a9-5c547fbb404a">

**Reply by katspaugh** [reply]
Byte range requests in and of themselves are just a way to seek in an audio track.
I assume you mean some sort of streaming/on demand partial loading of audio. It’s covered in the FAQ.
